CHANGELOG.md

```diff
@@ -1,19 +1,114 @@
+
+#### Device Trust for Linux support
+
+Teleport Device Trust now supports TPM joining on Linux devices.
+
+Additionally, `tsh proxy app` can now solve device challenges, allowing users to
+enforce the use of a trusted device to access applications.
+
+#### SSH connection resumption
+
+Teleport v15 introduces automatic SSH connection resumption if the network path
+between the client and the Teleport node is interrupted due to connectivity
+issues, and transparent connection migration if the control plane is gracefully
+upgraded.
+
+The feature is active by default when a v15 client (`tsh`, OpenSSH or PuTTY
+configured by `tsh config`, or Teleport Connect) connects to a v15 Teleport
+node.
+
+#### RDS auto-discovery in Access Management UI
+
+Users going through the Access Management UI flow to enroll RDS databases are
+now able to set up auto-discovery.
+
+#### EKS Integration for Teleport
+
+Teleport now allows users to enroll EKS clusters via the Access Management UI.
+
+#### Improved SAML IdP configuration flow
 
-#### FIPS now supported on ARM64
+When adding a SAML application via Access Management UI, users are now able to
+configure attribute mapping and have Teleport fetch service provider's entity
+descriptor automatically.
 
-Teleport 15 now provides FIPS-compliant Linux builds on ARM64. Users will now
-be able to run Teleport in FedRAMP/FIPS mode on ARM64.
+#### Improved provisioning for Okta
 
-#### Hardened AMIs now produced for ARM64
+Teleport 15 improves performance of receiving user/group updates from Okta by
+leveraging System for Cross-domain Identity Management (SCIM).
 
-Teleport 15 now provides hardened AWS AMIs on ARM64.
+Note: This feature will come out in a later 15.0 patch release.
 
-#### Streaming session playback
+#### Support for AWS KMS
+
+Teleport 15 supports the use of AWS Key Management Service (KMS) to store and
+handle the CA private key material used to sign all Teleport-issued
+certificates. When enabled, private key material never leaves AWS KMS.
+
+To migrate existing clusters to AWS KMS, you must perform a CA rotation.
+
+Docs:
+https://goteleport.com/docs/ver/15.x/choose-an-edition/teleport-enterprise/aws-kms/.
+
+#### MFA for administrative actions
+
+When Teleport is configured to require webauthn (`second_factor: webauthn`),
+administrative actions performed via `tctl` or the web UI will require an
+additional MFA tap.
+
+Examples of administrative actions include, but are not limited to:
+
+- Resetting or recovering user accounts
+- Inviting new users
+- Updating cluster configuration resources
+- Creating and approving access requests
+- Generating new join tokens
+
+Note: when MFA for administrative actions is enabled, certificates produced with
+`tctl auth sign` will no longer be suitable for automation due to the additional
+MFA checks. We recommend using Machine ID to issue certificates for automated
+workflows, which uses role impersonation that is not subject to MFA checks.
```

Comment:
nit: we might want to note that legacy clients that use certificates generated directly on the Auth server are not subject to MFA checks.

```suggestion
Note: when MFA for administrative actions is enabled, user certificates produced
with `tctl auth sign` will no longer be suitable for automation due to the additional
MFA checks, unless run directly on a local Auth server (legacy setup). We
recommend using Machine ID to issue certificates for automated workflows, which 
uses role impersonation that is not subject to MFA checks.
```

Reply:
The more I think about it, I think it is ok to leave this out of the release notes. I will make a note of it in the docs though.